nebius/terraform/infra/database.tf

```
@@ -0,0 +1,26 @@
+resource "nebius_msp_postgresql_v1alpha1_cluster" "metaflow_database_server" {
+  parent_id = var.project_id
+  name = var.database_server_name
+  bootstrap = {
+    db_name = "metaflow"
+    user_name = var.metaflow_database_server_admin_login
+    user_password = var.metaflow_database_server_admin_password
+  }
+  config = {
+    resources = {
```

```suggestion
    template = {
```

I had to change this to make it work with the newer version of nebius provider